ModPatches/Moyo 2/Defs/Moyo 2/Ammo/Ammo_Moyo_Industrial.xml

```diff
@@ -0,0 +1,210 @@
+		<description>Metal harpoons modified for launching in a spearjack.</description>
+		<statBases>
+			<Mass>2</Mass>
+			<Bulk>1.21</Bulk>
+		</statBases>
+		<tradeTags>
+			<li>CE_AutoEnableTrade</li>
+			<li>CE_AutoEnableCrafting_FabricationBench</li>
+		</tradeTags>
+		<thingCategories>
+			<li>AmmoLargeMoyoHarpoon</li>
+		</thingCategories>
+		<stackLimit>250</stackLimit>
+	</ThingDef>
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="LargeMoyoHarpoonBase">
+		<defName>Ammo_LargeMoyoHarpoon</defName>
+		<label>Heavy Spearjack</label>
+		<graphicData>
+			<texPath>Things/Ammo/Railgun/HighCaliber</texPath>
+			<graphicClass>Graphic_StackCount</graphicClass>
+		</graphicData>
+		<statBases>
+			<MarketValue>7.94</MarketValue>
+		</statBases>
+		<ammoClass>MoyoHarpoon</ammoClass>
+	</ThingDef>
+
+	<!-- ================== Projectiles ================== -->
+
+	<ThingDef Name="BaseSmallMoyoHarpoonBullet" ParentName="BaseBulletCE" Abstract="true">
+		<graphicData>
+			<texPath>Weapon/MoyoSpear</texPath>
+			<graphicClass>Graphic_Single</graphicClass>
+			<drawSize>(1.0,1.0)</drawSize>
+		</graphicData>
+		<projectile Class="CombatExtended.ProjectilePropertiesCE">
+			<damageDef>Bullet</damageDef>
+			<speed>40</speed>
+		</projectile>
+	</ThingDef>
+
+	<ThingDef ParentName="BaseSmallMoyoHarpoonBullet">
+		<defName>Bullet_SmallMoyoHarpoon</defName>
+		<label>Spearjack harpoon</label>
+		<projectile Class="CombatExtended.ProjectilePropertiesCE">
+			<damageAmountBase>26</damageAmountBase>
+			<armorPenetrationSharp>9</armorPenetrationSharp>
+			<armorPenetrationBlunt>100</armorPenetrationBlunt>
+		</projectile>
+	</ThingDef>
+	
+	<ThingDef Name="BaseLargeMoyoHarpoonBullet" ParentName="BaseBulletCE" Abstract="true">
+		<graphicData>
+			<texPath>Weapon/MoyoHarpoon</texPath>
+			<graphicClass>Graphic_Single</graphicClass>
+			<drawSize>(1.5,1.5)</drawSize>
+		</graphicData>
+		<projectile Class="CombatExtended.ProjectilePropertiesCE">
+			<damageDef>Bullet</damageDef>
+			<speed>40</speed>
+		</projectile>
+	</ThingDef>
+
+	<ThingDef ParentName="BaseLargeMoyoHarpoonBullet">
+		<defName>Bullet_LargeMoyoHarpoon</defName>
+		<label>Heavy Spearjack harpoon</label>
+		<projectile Class="CombatExtended.ProjectilePropertiesCE">
+			<damageAmountBase>75</damageAmountBase>
+			<armorPenetrationSharp>18</armorPenetrationSharp>
+			<armorPenetrationBlunt>800</armorPenetrationBlunt>
+		</projectile>
+	</ThingDef>
+
+	<!-- ==================== Recipes ========================== -->
+
+	<RecipeDef ParentName="AmmoRecipeBase">
+		<defName>MakeAmmo_SmallMoyoHarpoon</defName>
+		<label>make Spearjack x100</label>
+		<description>Craft 100 Spearjack.</description>
+		<jobString>Making Spearjack.</jobString>
+		<ingredients>
+			<li>
+				<filter>
+					<thingDefs>
+						<li>Steel</li>
+					</thingDefs>
+				</filter>
+				<count>50</count>
+			</li>
+		</ingredients>
+		<fixedIngredientFilter>
+			<thingDefs>
+				<li>Bioferrite</li>

```

Comment:
I'm not sure exactly why Bioferrite is in the recipe, but it doesn't exist if Anomaly isn't loaded, so we'll need make allowances for that. Could use `MayRequire="Ludeon.RimWorld.Anomaly"`, perhaps.